webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -38,6 +39,29 @@ function Drafts({
 }: Props) {
     const dispatch = useDispatch();
     const {formatMessage} = useIntl();
+    const [activeTab, setActiveTab] = useState<number>();
```

Comment:
Can't we infer the active tab from the route match? Something like...
```
const activeTab = isScheduledPostsTab ? 1 : 0;
```

Reply:
Done. We can, but I was going the more generic route to allow more tabs, but since we're not doing that now in other places, this can be removed as well.